test/tomo/geometry/spect_geometry_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test ODL geometry objects for SPECT."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+
+standard_library.install_aliases()
+
+# External
+import pytest
+import numpy as np
+
+# Internal
+import odl
+from odl.util.testutils import all_equal
+from odl.tomo.geometry.spect import ParallelHoleCollimatorGeometry
+
+
+def test_spect():
+    det_nx_pix = 64
+    det_ny_pix = 64
+    det_nx_mm = 4
+    det_radius = 200
+    n_proj = 180
+    det_param = det_nx_mm * det_nx_pix
+    dpart = odl.uniform_partition([-det_param, -det_param],
+                                  [det_param, det_param],
+                                  [det_nx_pix, det_ny_pix])
+
+    apart = odl.uniform_partition(0, 2 * np.pi, n_proj, nodes_on_bdry=True)
```

Comment:
Well, since this test does not use the SPECT data header, we may was well go for the more simple approach of skipping `nodes_on_bdry`. For an actual example (in the examples folder) where we use this with spect, we would indeed need to do this.